src/tactic/localized.lean

```diff
@@ -53,17 +53,15 @@ do m ← localized_attr.get_cache,
    return (ns.bind $ λ nm, m.find nm)
 
 /-- Execute all commands in the given notation namespace -/
-@[user_command] meta def open_locale_cmd (meta_info : decl_meta_info)
```

Comment:
Just to explain this at one point: there is an important semantic difference between having a `decl_meta_info` and not having one.  If there is a `decl_meta_info` argument, then docstrings (etc.) are possible on the user command, otherwise you get an error if you put a docstring above `open_locale`.

In this case, we don't want docstrings so it's ok to remove the argument.